Lib/idlelib/colorizer.py

```diff
@@ -27,12 +42,31 @@ def make_pat():
     sq3string = stringprefix + r"'''[^'\\]*((\\.|'(?!''))[^'\\]*)*(''')?"
     dq3string = stringprefix + r'"""[^"\\]*((\\.|"(?!""))[^"\\]*)*(""")?'
     string = any("STRING", [sq3string, dq3string, sqstring, dqstring])
-    return (kw + "|" + builtin + "|" + comment + "|" + string +
-            "|" + any("SYNC", [r"\n"]))
+    prog = re.compile("|".join([
+        builtin, comment, string, kw,
+        match_softkw, case_softkw_and_pattern,
+        any("SYNC", [r"\n"]),
+    ]), re.DOTALL | re.MULTILINE)
```

Comment:
```suggestion
    prog = re.compile("|".join([
                                builtin, comment, string, kw,
                                match_softkw, case_softkw_and_pattern,
                                any("SYNC", [r"\n"])
                              ]),
                      re.DOTALL | re.MULTILINE)
```
Putting arguments on a totally dedented line seems wrong.  In any case, formatting this call with nested calls, one with a list arg,  is awkward.  My suggestion is clearer to me as better indicating which items are part of what.

Reply:
Done.